### PR TITLE
Change all references from fedorahosted.org to use the github area.

### PR DESCRIPTION
**Pull request by nirik**
Note that we will have to change this all again when we move to pagure.

### CODE REVIEW

File: doc/contributing.rst [hunk 1]
```diff
@@ -15,5 +15,4 @@ If you're submitting patches to MirrorManager2, please observe the following:
    idea but don't know how to implement it, you just have something bugging
    you?
 
-   Come to see us on IRC: ``#fedora-apps`` on irc.freenode.net or via the
-   `trac of the project <http://fedorahosted.org/packagedb/>`_.
```
Comment (pypingou) [inline doc/contributing.rst]: oh, nice one :D

**Comment by pypingou** [comment]
👍 for me

**Comment by adrianreber** [comment]
Nice, thanks.

**Comment by puiterwijk** [comment]
Headsup: #214 reverts part of this PR, since changing namespace suddenly is problematic.

**Pull request merged**